tests/test_speed.py

```diff
@@ -20,6 +20,20 @@ def test_SpeedAndBufferSet(self, dvs, testlog):
         cdb = dvs.get_config_db()
         adb = dvs.get_asic_db()
 
+        # Get speed from the first port we hit in ASIC DB port walk, and
+        # assume that its the initial configured speed for all ports, and
+        # dynamic buffer profile has already been created for it.
+        asic_port_records = adb.get_keys("ASIC_STATE:SAI_OBJECT_TYPE_PORT")
+        for k in asic_port_records:
+            fvs = adb.get_entry("ASIC_STATE:SAI_OBJECT_TYPE_PORT", k)
+            for fv in fvs.keys():
+                if fv == "SAI_PORT_ATTR_SPEED":
+                    configured_speed_list.append(fvs[fv])
+                    break
```

Comment:
```suggestion
            if "SAI_PORT_ATTR_SPEED" in fvs:
                configured_speed_list.append(fvs[fv])
```
Since entries are `dicts` we don't need to loop through to see if the attribute exists.

Reply:
Modifies as requested.